dist/package.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
any idea why this adds a package.json file to the dist folder?

Reply:
Not sure! I will check across revisions and see what causes this.